README.md

```diff
@@ -224,6 +224,17 @@ If [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy), the modul
 
 If a value of `'minimal'` is set, the progress indicator will render as a small, colored bar at the top of the window. This can be useful when the default fancy progress indicator interferes with elements in the page.
 
+### `ramdisk`
+Type: `boolean`<br>
+Default: `false`<br>
+Support: MacOS and Linux, Windows with WSL 2.0.
+
+If `true`, will apply [`webpack-plugin-ramdisk`](https://www.npmjs.com/package/webpack-plugin-ramdisk) to the build. `output` configuration does not have to be modified, a symlink will be created from the original output path to the output path on the ramdisk.
```

Comment:
Maybe we should make the removal of existing `output.path` here clearer, something like a warning in case someone doesn't want `output.path` being removed on every build.

Reply:
I think a console warning would be too chatty for 99% of people using this. I can make it more obvious in the README, but that'd be about the extent to which I'd go. Users will already be used to webpack overwriting files in the directory for each build. There's not much functional difference here.

Reply:
Agreed. I was talking about a warning in the README.

Reply:
Please check the latest commit.